test/tools/Modules/HelpersCommon/HelpersCommon.psd1

```diff
@@ -12,7 +12,7 @@ GUID = 'cc1c8e94-51d1-4bc1-b508-62bc09f02f54'
 
 CompanyName = 'Microsoft Corporation'
 
-Copyright = 'Copyright (C) Microsoft Corporation, All rights reserved.'
+Copyright = 'Copyright (c) Microsoft Corporation, All rights reserved.'
```

Comment:
Please replace "," with dot.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."